src/main/java/com/google/api/codegen/config/SampleConfig.java

```diff
@@ -50,14 +99,110 @@ public static Builder newBuilder() {
   @AutoValue.Builder
   public abstract static class Builder {
 
+    public abstract Builder id(String val);
+
+    public abstract Builder title(String val);
+
+    public abstract Builder description(String val);
+
+    public abstract Builder interfaceConfig(InterfaceConfig val);
+
+    public abstract Builder methodConfig(MethodConfig val);
+
+    public abstract Builder requestConfigs(ImmutableList<RequestFieldProto> val);
+
+    public abstract Builder responseConfigs(ImmutableList<ResponseStatementProto> val);
+
     public abstract Builder regionTag(String val);
 
     public abstract Builder callingForm(CallingForm val);
 
+    public abstract Builder callingPattern(String val);
+
     public abstract Builder valueSet(SampleValueSet val);
 
     public abstract Builder type(SampleSpec.SampleType val);
 
     public abstract SampleConfig build();
   }
+
+  public static ImmutableList<SampleConfig> create(
```

Comment:
I would rename this to `createList` or `createMultiple` to avoid confusion with the overload. They're related but do different things, so they should have different names,

Reply:
Changed this to `createSampleConfigs`, and other one to `createSampleConfig`.